src/components/EditContentModal/EditContentModal.tsx

```diff
@@ -5,12 +5,12 @@ import {
   FormGroup,
   Modal,
   ModalVariant,
-  Popover,
+  Popover, Radio,
```

Comment:
A linter addon would have caught this.

Reply:
I did a thing 😆

Reply:
It was format you needed to run in this case.

Reply:
I have prettier and eslint as addons for this reason. 
They work together to point out and automagically formatting issues. 
The only reason I'm a stickler is it makes later merging/rebasing easier.

Reply:
what's the command for that?